pkgs/development/libraries/azure-sdk-for-cpp/core-amqp.nix

```
@@ -0,0 +1,78 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  core,
+  c-shared-utility,
+  macro-utils-c,
+  umock-c,
+  nix-update-script,
+  meta,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "azure-sdk-for-cpp-core-amqp";
+  version = "1.0.0-beta.11";
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "Azure";
+    repo = "azure-sdk-for-cpp";
+    tag = "azure-core-amqp_1.0.0-beta.11";
+    hash = "sha256-MQsz5Dmv1BwfUaN1VXMC3hPdMHihlgOBaukp5wgTNJc=";
+  };
+  sourceRoot = "source/sdk/core/azure-core-amqp";
+
+  postPatch = ''
+    sed -i '/CMAKE_CXX_STANDARD/d' CMakeLists.txt
+  '';
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+  ];
+
+  propagatedBuildInputs = [
+    core
+    c-shared-utility
+    macro-utils-c
+    umock-c
+  ];
+
+  env = {
+    AZURE_SDK_DISABLE_AUTO_VCPKG = 1;
+    NIX_CFLAGS_COMPILE = "-Wno-error";
+  };
+
+  cmakeFlags = [
+    "-DBUILD_SHARED_LIBS=ON"
+    "-DDISABLE_RUST_IN_BUILD=ON"
+    "-DWARNINGS_AS_ERRORS=OFF"
+  ];
+
+  postInstall = ''
+    moveToOutput "share" $dev
```

This line also moves `share/azure-core-amqp-cpp/copyright` to `$dev`. If a dependent package's output content only references `azure-sdk-for-cpp.core-amqp.out` but not `azure-sdk-for-cpp.core-amqp.dev`, the `copyright` file won't be downloaded to the users' machine, which violates the MIT license this package is distributed under.

```suggestion
    moveToOutput "share" "$dev"
    moveToOutput "share/azure-core-amqp-cpp/copyright" "$out"
```